### ISSUE TITLE
Design images for social media

### ISSUE THREAD

**Issue by FelixHenninger** [FelixHenninger/lab.js#14]
**@katjaq has asked for a nice illustration of our project for sharing on social media.** That's a fantastic idea, but we don't have a go-to-option that represents our project well. _Let's make one!_

**How you can help**
* Any ideas, thoughts and feedback on the material are super-welcome!
* Please be warmly invited to doodle, draw or design, any contribution is hugely appreciated!

**Comment by vsoch** [comment]
This is right on, and I'd again say to keep the current logo (but in a few different styles and formats!). Another thing I noticed is that the logo isn't prominently on the experiment builder page, might be nice to add it subtly somewhere!

**Comment by FelixHenninger** [comment]
Hej, thanks! I'm aching for a few free days to make some more fancy graphics -- did you have any specific styles in mind? I also like the logo, but I'm a bit out of ideas as to what to do with it, and the yellow is getting a bit old.

**Comment by FelixHenninger** [comment]
Oh, and regarding the experiment builder, did you have the graphic above in mind, or the actual interface? It's there in the bottom left corner, very well hidden clearly 😉.

**Comment by vsoch** [comment]
ah yes you are right! Maybe it's ok then :) I just never remember seeing it (and in fact I didn't, heh).

**Comment by FelixHenninger** [comment]
I had the opposite take-away, that it should be given more space :). I'll keep that at the back of my mind -- it used to be huge and get in the way, and we might have over-corrected.
